### DISCUSSION TITLE
Team 4 Lab 1

### DISCUSSION THREAD

**Discussion by twangss** [atsa-es/fish550-2023#8] [General]
Report what you work on and when @veggerk

**Comment by veggerk** [comment]
working on stationarity tests

**Reply by twangss** [reply]
cool thanks! illl fit arima models

**Reply by veggerk** [reply]
OK, I just pushed so you should be good to go.

**Reply by twangss** [reply]
@veggerk heyo, i just pushed hw1 changes. I think we are v close to done, do you mind reading it over to make sure if it makes sense?

**Reply by veggerk** [reply]
I read through it, I think it looks good!

**Reply by twangss** [reply]
submitted!
